pySTEL/boozer_util.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Main routine
+if __name__=="__main__":
+	import sys
+	from argparse import ArgumentParser
+	import numpy as np
+	import matplotlib.pyplot as pyplot
+	from libstell.boozer import BOOZER
+	parser = ArgumentParser(description= 
+		'''Provides class for accessing boozer data also servers as a
```

Comment:
"also servers" -> "and also serves"

Reply:
This typo is repeated in at least one other pySTEL file, probably check them all.